yotta/data/entropy_hardware_poll.c

```diff
@@ -1,88 +0,0 @@
-/*
- *  Hardware entropy collector for the K64F, using Freescale's RNGA
```

Comment:
Does Mbed OS 5 have one of these hardware entropy collector thingies?

Reply:
It does. [Here](https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/trng_api.c) is the one for K64F. However, it works differently from the one here because Mbed OS has its own TRNG API.